polaris.shopify.com/src/components/ComponentExamples/ComponentExamples.tsx

```diff
@@ -81,61 +85,172 @@ const ComponentExamples = ({examples}: Props) => {
     return () => clearInterval(waitForExampleContentToRender);
   };
 
+  // get example names and index
+  const defaultExampleName = examples[0].fileName.replace('.tsx', '');
+  let exampleName = params.get('example') || defaultExampleName;
+  let exampleFilename =
+    (exampleName ? exampleName : defaultExampleName) + '.tsx';
+  let exampleIndex = examples.findIndex(
+    (obj) => obj.fileName === exampleFilename,
+  );
+
+  // update query param to the selected example
+  const updateSelectedExample = (index: number) => {
+    exampleName = examples[index].fileName.replace('.tsx', '');
+
+    router.replace(
+      {
+        pathname: router.pathname,
+        query: {...router.query, example: exampleName},
+      },
+      undefined,
+      {shallow: true},
+    );
+  };
+
+  // manage keyboard focus for tabs and tabpanels
```

Comment:
The following will allow you to navigate the tabs using `tab` `left arrow` `right arrow` `Home` and `End` keys